source/_integrations/matter.markdown

```diff
@@ -167,6 +167,11 @@ Once the initial firmware upgrade to Matter is complete, the device can also be
 - [Eve Door & Window on Amazon](https://amzn.to/3RIU6ml)
 - [Eve Motion on Amazon](https://amzn.to/3jDujiP)
 
+### Zemismart Roller Shade Motor (cover)
```

Comment:
this almost feels like a commercial for this hardware. Maybe we should just go for a device compatibility table in a next PR where this device is also mentioned. For now, either include (neutral) device specific instructions or leave out this text. Thanks!